strutil/strutil.go

```diff
@@ -40,3 +45,30 @@ func JSON(v interface{}) string {
 	bytes, _ := json.Marshal(v)
 	return string(bytes)
 }
+
+func wildCardToRegexp(pattern string) string {
+	patternRegex := regexp.QuoteMeta(pattern)
+	patternRegex = strings.Replace(patternRegex, "\\?", ".", -1)
+	patternRegex = strings.Replace(patternRegex, "\\*", ".*", -1)
+	return patternRegex
+}
+
+func regexMatch(pattern string, value string) bool {
+	result, _ := regexp.MatchString(pattern, value)
+	return result
+}
+
+// IsURLExcluded checks whether given urlPath matches any of the exclude patterns.
+func IsURLExcluded(srcurl *url.URL, excludeUrls []*url.URL) bool {
```

Comment:
What are your thoughts on moving this function to `command` package, like `expandSource` and making it unexported?

Reply:
moved it to newly created `command/exclude.go`.